README.md

```diff
@@ -1,7 +1,62 @@
-# codeartifact-workflows
-Public re-usable GitHub workflows for working with AWS CodeArtifact
+# python-lib-workflows
+Public re-usable GitHub workflows for working with python libraries which autorelease with bumpversion and publish to AWS CodeArtifact
 
 # Usage
```

Comment:
```suggestion

### How does it do it?

# Usage
```

Reply:
I don't understand this change... is this a section you want to live under `##What is it?`?

Might be cleaner to branch and raise a PR into this branch rather than use the suggestions if you want to make structural changes like this. :-)